security/mobile-app-network-security.md

```diff
@@ -0,0 +1,120 @@
+# Abstract
+
+In this document we will discuss some of the common problems around network security for mobile developers consuming AeroGear services in their apps, and propose a solution that will help resolve those problems.
+
+We will first look at some of the use cases in order to define the problems, and then look at some of the options to solve them. Finally the proposed solution will be presented.
+
+# Goals
+
+## Primary Goal
+
+* Reach agreement on a solution that can help mobile developers properly handle network security in their mobile apps that consume AeroGear services
```

Comment:
Just to confirm. This is only for network security between the app and AeroGear services. Not between an app and any other stuff it communicates with?

Reply:
yes, that's right.